### PR TITLE
Release 1.3.1

### PR DESCRIPTION
**Pull request by ChrisAdderley**

### CODE REVIEW

File: Distribution/Restock/GameData/ReStock/Patches/Electrical/restock-lights.cfg [hunk 1]
```diff
@@ -69,7 +69,7 @@
 }
 
 // Illuminator Mk2
-@PART[spotLight2]:HAS[~RestockIgnore[*]]:FOR[000_ReStock]
+@PART[spotLight2_v2]:HAS[~RestockIgnore[*]]:FOR[000_ReStock]
```
Comment (drewcassidy) [inline Distribution/Restock/GameData/ReStock/Patches/Electrical/restock-lights.cfg:72]: I don't think this should be applied to spotLight2_v2 twice?

**Review by Poodmund — APPROVED** [review]
All seems fine. Gigantor issue is resolved enough to a patchable degree for release.

**Review by Poodmund — COMMENTED** [review]
Issue with the RV-105-A ReStock+ part, see: https://github.com/PorktoberRevolution/ReStocked/issues/942

**Review by Poodmund — APPROVED** [review]
Just taken a look at the latest commit, seems good now.

**Pull request merged**